packages/ocaml-base-compiler/ocaml-base-compiler.4.10.1+macos-arm64/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "Release 4.10.1 with macOS/arm64 support"
+maintainer: "platform@lists.ocaml.org"
+authors: "Xavier Leroy and many contributors"
+homepage: "https://ocaml.org"
+bug-reports: "https://github.com/ocaml/ocaml/issues"
+dev-repo: "git://github.com/ocaml/ocaml"
+depends: [
+  "ocaml" {= "4.10.1" & post}
+  "base-unix" {post}
+  "base-bigarray" {post}
+  "base-threads" {post}
+]
+conflict-class: "ocaml-core-compiler"
+flags: compiler
+setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
+build: [
+  [
+    "./configure"
+    "--prefix=%{prefix}%"
+    "-C"
+    "CC=cc" {os = "openbsd" | os = "freebsd" | os = "macos"}
+    "ASPP=cc -c" {os = "openbsd" | os = "freebsd" | os = "macos"}
+  ]
+  [make "-j%{jobs}%" {os != "cygwin"} "world"]
+  [make "-j%{jobs}%" {os != "cygwin"} "world.opt"]
+]
+install: [make "install"]
+url {
+  src: "git+https://github.com/avsm/ocaml.git#4.10.1+macos-arm64"
```

Comment:
There is no corresponding path on ocaml’s main compiler branch?

Reply:
Should we also add the md5/sha?

Reply:
I believe this is the same sort of issue as with https://github.com/ocaml/opam-repository/pull/16357

Reply:
Yeah, git remote is fine for this sort of "temporary" package (since 4.12 will supplant it with an official release). Good point about moving it to ocaml/ though rather than my fork -- I'll talk to @Octachron about that.